content/en/automation.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Definition Template
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+
+Automation is a term for technology applications where human input is minimized.
+When any process is set-up in a way which requires less or zero human intervention is known as automation.
+
+## Problem it addresses 
+
+Automation helps administrators and operators manage environments by automating previously manual operations.
+
+## How it helps
+
+Automation speeds up administration and configuration of environments thus requiring less man power
```

Comment:
I think concepts omit the various sections and just describe the thing. Look at Abstraction as an example. Regardless, we'll need to avoid any gendered language. Swap "man power" for something more neutral like, "engineering time".

Reply:
Sir considering your suggestion, I have made few changes 
I have also tried explaining the term, with the help of example
Kindly check and suggest more, if anything else is required 
thank you :)